jekyll/_cci2/next-gen-migration-guide.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: classic-docs
+title: "Migrating to next-gen Convenience Images"
+short-title: "Migrating to next-gen Convenience Images"
+description: "Migrating to next-gen Convenience Images"
+order: 30
+version:
+- Cloud
+- Server v2.x
+---
+
+* TOC
+{:toc}
+
+
+## Overview
+
+Last year (2020) CircleCI introduced the next generation (next-gen) of Convenience Images. These new images are designed to replace the legacy Convenience Images that came out during the announcement of CircleCI 2.0, about 4 years ago. The next-gen CircleCI Convenience Images are designed from the ground up for a Continuous Integration environment. They’re designed to be faster, more efficient, and more importantly, more reliable. You can learn more about all of the features on our blog post. As we begin to deprecate the legacy images, here is information on the migration process.
```

Comment:
```suggestion
In 2020 CircleCI introduced the next generation (next-gen) of convenience images. These new images are designed to replace the legacy convenience images that came out during the announcement of CircleCI 2.0, around 2016. The next-gen CircleCI convenience images are designed from the ground up for a CI/CD environment. These images are designed to be faster, more efficient, and, most importantly, more reliable. You can learn more about these features in our blog post. As we begin to deprecate the legacy images, this page outlines information on the migration process.
```

Reply:
@felicianotech can we link to the blog post here?

Reply:
There two links/blog posts here. I added both back.